resources/frontend_client/index_template.html

```diff
@@ -41,6 +41,7 @@
 
     <!-- If you modify this script, make sure you update the whitelisted Content-Security-Policy hash in metabase.server.middleware.security -->
     <script type="text/javascript">{{{bootstrapJS}}}</script>
+    <script type="text/javascript">{{{assetOnErrorJS}}}</script>
```

Comment:
I wonder if we would need to modify `inline-js-hashes` there to add `frontend_client/inline_js/asset_loading_error.js` to this block?

```
    (mapv file-hash [ ;; inline script in index.html that sets `MetabaseBootstrap` and the like
                     "frontend_client/inline_js/index_bootstrap.js"
                     ;; inline script in init.html
                     "frontend_client/inline_js/init.js"])))
```

Reply:
Yes! @npfitz please check this is Safari as it's more strict about CSP.